src/rgw/rgw_sal_motr.h

```diff
@@ -360,7 +360,7 @@ class MotrBucket : public Bucket {
     virtual int set_acl(const DoutPrefixProvider *dpp, RGWAccessControlPolicy& acl, optional_yield y) override;
     virtual int load_bucket(const DoutPrefixProvider *dpp, optional_yield y, bool get_stats = false) override;
     int link_user(const DoutPrefixProvider* dpp, User* new_user, optional_yield y);
```

Comment:
Should we do similar changes for link_user method ? This will make it consistent
cc - @bkirunge7  @jjxsg

Reply:
We can change it, although it will increase the scope of testing for this ticket as all the create bucket scenario must be tested after changing `link_user()`

Reply:
This can be done later